dutch/over-het-hek/hek.md

```diff
@@ -0,0 +1,5 @@
+... vol overgave neem je een aanloop en springt over het hek. Met een elegante zwaai vlieg je eroverheen.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.